app/modules/Documentation/services/Article.php

```diff
@@ -0,0 +1,236 @@
+<?php
+
+/**
+ * This file is part of Vegas package
+ *
+ * @author Tomasz Borodziuk <tomasz.borodziuk@amsterdam-standard.nl>
+ * @copyright Amsterdam Standard Sp. Z o.o.
+ * @homenavigation https://bitbucket.org/amsdard/vegas-phalcon
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Documentation\Services;
+
+use Documentation\Models\Article as ArticleModel;
+use Phalcon\DI\InjectionAwareInterface;
+use Vegas\DI\InjectionAwareTrait;
+
+class Article implements InjectionAwareInterface
+{
+    use InjectionAwareTrait;
+    
+    private $serviceVersion = null;
+    private $serviceCategory = null;
+
+    private function getServiceVersion()
+    {
+        if($this->serviceVersion === null) {
+            $this->serviceVersion = $this->di->get('serviceManager')->get('documentation:version');
+        }
+        return $this->serviceVersion;
+    }
+    
+    private function getServiceCategory()
+    {
+        if($this->serviceCategory === null) {
+            $this->serviceCategory = $this->di->get('serviceManager')->get('documentation:category');
+        }
+        return $this->serviceCategory;
+    }
+    
+    public function updateContent($id, $content, $contentRendered, $archival=false)
+    {
+        $article = $this->getArticle($id);
+        if(!empty($article)) {
+            $article->content = $content;
+            $article->contentRendered = $contentRendered;
+            if($archival) {
+                return $article->makeCopyToArchive(0);
+            }
+            return $article->save();
+        }    
+        return false;
+    }
+    
+    public function updatePosition($id, $position)
+    {
+        $article = $this->getArticle($id);
+        if(!empty($article)) {
+            $article->position = $position;
+            return $article->save();
+        }
+        return false;
+    }
+    
+    public function getArticle($id)
+    {        
+        $object = ArticleModel::findById($id);
+        return $object;
+    }
+    
+    public function getArticleBySlug($versionSlug,$categorySlug,$articleSlug,$onlyPublished=true)
```

Comment:
add spaces in method parameters, we are coding with PSR-2 standards:
https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md
please also check in other methods